packages/passport/sdk/src/utils/string.ts

```diff
@@ -5,8 +5,7 @@ export const hexToString = (hex: string) => {
 
   try {
     const stripped = utils.stripZeros(utils.arrayify(hex));
-    const buff = Buffer.from(stripped);
-    return buff.length === 32 ? hex : utils.toUtf8String(stripped);
+    return stripped.length === 32 ? hex : utils.toUtf8String(stripped);
```

Comment:
@Jon-Alonso @haydenfowler  Can someone confirm what the `length === 32` check is for here?

Reply:
Thanks for the info. We can probably just remove the check then if other implementations result in the same behaviour.